spec/rubocop/cop/ezcater/feature_flag_active_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   before { inspect_source(line) }
 
-  %w(EzcaterFeatureFlag EzFF).each do |constant_name|
+  %w(EzcaterFeatureFlag EzFF ::EzFF).each do |constant_name|
```

Comment:
🎱 Super unlikely that we'd need it, but is there value in being pedantic and adding `::EzcaterFeatureFlag` too while we're adding scoped options here?

Reply:
Probably. In all honestly neither really needs to be here. This is a relic from when I thought the namespacing was the issue, but those specs passed without needing to change anything, as evidenced by the eventual change I figured out. But I think there's some value in having these here as regression-proofing if Ruby AST shapes ever change.